src/main/java/com/ironcorelabs/tenantsecurity/kms/v1/SecurityEventResult.java

```diff
@@ -0,0 +1,19 @@
+package com.ironcorelabs.tenantsecurity.kms.v1;
+
+import com.google.api.client.util.Key;
+
+public class SecurityEventResult {
```

Comment:
Is there anything we could communicate back to the TSC from the TSP about a failure by way of TenantSecurityKMSException? It seems weird to have this type. A `true` semantically is a `()`, just that the call went through fine and TSP is going to process it. An `Exception` in the CF means that something went wrong (in this case probably just network errors or one of the normal TSC ones like wrong key). A `false` means something failed but we don't know what? It seems like we should be able to tell them by way of an error code/message if the TSP determined their data was invalid for ex.

If we can't, maybe the return type of the API calling function should still be `Void`/`()`, we throw an Exception on a `false` response and the good explanation of queuing goes on that client function?

Reply:
After thinking about this more, I think you're right. I kept expecting to find something else to put in there, I guess. The only reason I can think of to keep the result object would be for forward compatibility, but even that seems lackluster since the return type would just go from `void` to something. I will make the change here and in TSP.